api/dbadapters/bigquery.ts

```diff
@@ -91,22 +93,57 @@ export class BigQueryDbAdapter implements IDbAdapter {
       .promise();
   }
 
-  public async evaluate(statement: string) {
-    try {
-      await this.getClient().query({
-        useLegacySql: false,
-        query: statement,
-        dryRun: true
-      });
-      return dataform.QueryEvaluationResponse.create({
-        status: dataform.QueryEvaluationResponse.QueryEvaluationStatus.SUCCESS
-      });
-    } catch (e) {
-      return dataform.QueryEvaluationResponse.create({
-        status: dataform.QueryEvaluationResponse.QueryEvaluationStatus.FAILURE,
-        error: parseBigqueryEvalError(e)
-      });
+  public async evaluate(
+    query: string | dataform.Table | dataform.Operation | dataform.Assertion,
+    projectConfig?: dataform.IProjectConfig
```

Comment:
AFAICT this is actually required now, otherwise the code won't work? seems like it should be required here and in the interface

Reply:
Yeah pretty weird that that's allowed. Have updated to use new interface in other adapters, and added a throw if they're given things other than strings.